python/ray/_private/gcs_pubsub.py

```diff
@@ -48,31 +49,107 @@ def construct_error_message(job_id, error_type, message, timestamp):
     return data
 
 
+class _PublisherBase:
+    def _create_log_request(self, log_batch: dict):
```

Comment:
Could we move logging related code into a file like gcs_pubsub_logs.py?

Reply:
Moved dict <-> LogBatch proto conversions to logging_utils.py. LMK if this is what you have in mind.